docs/roman/pipeline/calroman_level2.rst

```diff
@@ -0,0 +1,81 @@
+.. _calroman_level2:
+
+
+calroman_level2: Stage 2 Detector Processing
+==============================================
+
+:Class: `romancal.pipeline.Level2Pipeline`
+:Alias: calroman_level2
+
+The ``Level2Pipeline`` applies basic detector-level corrections to given exposure
```

Comment:
I would remove `basic` from this sentence, not sure what it means in this context.